src/operator/gitea.ts

```diff
@@ -0,0 +1,161 @@
+import Operator from '@dot-i/k8s-operator'
+import * as k8s from '@kubernetes/client-node'
+import { KubeConfig } from '@kubernetes/client-node'
+import stream from 'stream'
+
+interface ConditionCheckResult {
+  ready: boolean
+  pod: k8s.V1Pod | undefined
+}
+
+const kc = new KubeConfig()
+// loadFromCluster when deploying on cluster
+// loadFromDefault when locally connecting to cluster
+kc.loadFromCluster()
+const k8sApi = kc.makeApiClient(k8s.CoreV1Api)
+
+function buildTeamString(teamNames: any[]): string {
```

Comment:
Why do not you create a regular object and use JSON.stringify ?

Reply:
Agree, could be done better. I'll make changes, this way it will also be better readable.